clusterloader2/testing/load/experimental/overrides/enable_networkpolicies.yaml

```diff
@@ -0,0 +1 @@
+ENABLE_NETWORKPOLICIES: true
```

Comment:
You should add this knob in a separate PR, then enable it in perf-tests presubmit and the follow with the rest of this PR. This way, the presubmit will test the codepath/scenarios that's being added as part of this PR preventing breakages.

This has been documented in [here](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/experiments.md)

Reply:
This is an experimental suite, it's not part of any CI/CD tests. Therefore, I don't think that the rollout precedure applies here :)

Reply:
Make sense.